### PR TITLE
Added 'Duc' package (High-performance disk usage analyzer)

### PR DESCRIPTION
**Pull request by zevv**
http://duc.zevv.nl

```
       Duc is a collection of tools for inspecting and visualizing disk usage.

       Duc  maintains  an  indexed database of accumulated sizes of directories of your file
       system, and allows you to query this database with some tools, or create  fancy  sun‐
       burst graphs to show you where your bytes are.

       Duc scales quite well, it has been tested on systems with more then 500 million files
       and several petabytes of storage.
```

### CODE REVIEW

File: packages/duc/build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://www.stunnel.org/
```
Comment (joakim-noah) [inline packages/duc/build.sh]: Forgot to add your home page URL?

File: packages/duc/build.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+TERMUX_PKG_DESCRIPTION="High-performance disk usage analyzer"
+TERMUX_PKG_VERSION=1.4.3
+TERMUX_PKG_SHA256=504810a1ac1939fb1a70bd25e492f91ea38bcd58ae0a962ce5d35559d7775e74
+TERMUX_PKG_SRCURL=https://github.com/zevv/duc/releases/download/${TERMUX_PKG_VERSION}/duc-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_DEPENDS="leveldb, ncurses"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-x11 --with-db-backend=leveldb --disable-cairo"
+TERMUX_PKG_BUILD_IN_SRC="yes"
+TERMUX_PKG_RM_AFTER_INSTALL=""
```
Comment (joakim-noah) [inline packages/duc/build.sh]: This is to remove unnecessary files, you're not doing anything. Remove the last two empty lines too.

File: packages/duc/build.sh (hunk 1 above)
Comment (zevv) [reply packages/duc/build.sh]: Yeah, having fun with copy&pasting, sorry for that

File: packages/duc/build.sh (hunk 2 above)
Comment (zevv) [reply packages/duc/build.sh]: Ok

**Comment by zevv** [comment]
Updated in 020ce12a31937de3ebf81eda24c501a7431e32e2

**Comment by joakim-noah** [comment]
Ping @fornwall.

**Comment by zevv** [comment]
ping?

**Comment by xeffyr** [comment]
Package is now available in [unstable-packages](https://github.com/termux/unstable-packages) repository.
```
pkg install unstable-repo
pkg install duc
```

**Pull request closed without merging**